### ISSUE TITLE
Translation and design issue in import excel data frame dialog

### ISSUE THREAD

**Issue by MeSophie** [africanmathsinitiative/R-Instat#7767]
By using R-Instat 0.7.8, we have some words in Import excel data frame  (File>Import From File>Select excel file) **nombre de ligne à prévisualiser** and **nombre de ligne à ignorer** are superposed on the selection box. Also **Nombre maximal de lignes à** is not well translate (It is missing **ignorer**).

![image](https://user-images.githubusercontent.com/106779233/184326183-93a418a2-07e3-464d-9d8c-c4c9f7830eaf.png)

**Comment by N-thony** [comment]
@MeSophie  can you check with merged version if this is already fixed? I found that same of the translations issues you noticed during the DRC workshop with the first released version 0.7.8 are already fixed in the merged version. If the issue is already fixed in the merged version, please clode it. Thanks.

**Comment by MeSophie** [comment]
@N-thony  I assume that the merged version is the update version in VS. If Yes, there is still **Nombre maximum de ligne à** which is not well translate. Thank you.

**Comment by N-thony** [comment]
> @N-thony I assume that the merged version is the update version in VS. If Yes, there is still **Nombre maximum de ligne à** which is not well translate. Thank you.

@MeSophie yes, check the issues you have opened about the translation in the merged version, if the problem is still there leave the issue open otherwise close it.
